http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1613,6 +1613,281 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
+  "/proxy/{googleProject}/{runtimeName}/yarn":
```

Comment:
Looks like Swagger-UI alphabetizes endpoints so these get kind of mixed together:

![image](https://user-images.githubusercontent.com/5368863/127031121-332b5cb0-2d05-4d0d-83fc-05b0fecd223f.png)


I wonder if they should have a prefix like `/proxy/{googleProject}/{runtimeName}/spark/yarn`, `/proxy/{googleProject}/{runtimeName}/spark/hdfs`, etc?

Reply:
Another option may be to hyphenate like in `/proxy/../spark-yarn`, `/proxy/../spark-hdfs`, etc.

Reply:
I kind of like that convention